layers/generate_layers.py

```diff
@@ -99,8 +99,8 @@ def main():
         )
 
 
-# Extract `{tagPartOne}={tagPartTwo}` polygons from OSM, and only keep a name attribute.
-def generatePolygonLayer(osm_input, tagPartOne, tagPartTwo, filename):
+# Extract polygons from OSM using a tag filter, and only keep a name attribute.
```

Comment:
I don't really think it's helpful to split into two parameters. Whoever's calling this has to go read the osmium docs to figure out the format anyway

Reply:
I agree, nice change